ko/sql/jsp.rst

```diff
@@ -918,15 +919,17 @@ CUBRID의 Java SP에서는 JNI 기능을 사용할 수 있도록 제공하고 
 
 .. warning::
 
-    JNI를 호출하는 자바 저장 프로시저/함수 실행 시 java.lang.UnsatisfiedLinkError가 발생할 수 있다.
+    JNI를 호출하는 자바 저장 프로시저/함수를 **-j** 옵션 없이 등록한 후 실행 시 java.lang.UnsatisfiedLinkError가 발생할 수 있다.
```

Comment:
@hgryoo 
Please also revise the sentence of English manual.

Reply:
fixed at e81e1d9